packages/validation/src/validators/yaml-syntax/validator.ts

```diff
@@ -62,11 +62,16 @@ export class YamlValidator extends AbstractPlugin {
     const locations = createLocations(resource, region);
     const ruleId = YAML_RULE_MAP[err.code];
 
-    return this.isRuleEnabled( ruleId ) ? this.createValidationResult(ruleId, {
-      message: {
-        text: err.message,
-      },
-      locations,
-    }) : undefined;
+    const textWithoutLocation =
+      err.message.split(" at line").at(0) ?? err.message;
```

Comment:
It's important to add this error's message though as otherwise the error is extremely vague and unhelpful.

The error message contains a custom stacktrace which looks very bad so I remove that.